_posts/platform/deployment/2000-01-01-deploy-java-jar-war.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Deployment of JAR and WAR archives
-nav: JAR and WAR archives
-modified_at: 2018-02-09 00:00:00
+nav: Deploy JAR/WAR
+modified_at: 2020-02-18 00:00:00
```

Comment:
could you add in this file that you only support Tomcat as a destination app server ?
and point a link to the java buildpack in case of a fat jar ?

Reply:
> could you add in this file that you only support Tomcat as a destination app server ?

:+1: 

> and point a link to the java buildpack in case of a fat jar ?

Not sure I understand, can you elaborate?

Reply:
>> and point a link to the java buildpack in case of a fat jar ?

> Not sure I understand, can you elaborate?

Explain that if you have a fat jar you need to use the Buildpack which only provides a jdk (which is it again ? ^^)

Reply:
I don't know how to do that. Did you do it and you can share the commands you typed to do so?